src/AzureExtension/Helpers/LocalSettings.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Windows.Storage;
+
+namespace DevHomeAzureExtension.Helpers;
+
+public static class LocalSettings
+{
+    private static readonly string _applicationDataFolder = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "DevHome/ApplicationData");
+    private static readonly string _localSettingsFile = "LocalSettings.json";
+
+#pragma warning disable CA1859 // Use concrete types when possible for improved performance
```

Comment:
Why not follow the advice?

Reply:
I literally took this directly from https://github.com/microsoft/devhomegithubextension/blob/main/src/GitHubExtension/Helpers/LocalSettings.cs

I added the warning suppress to clean up the build to not have the superfluous warning. As for the reason it uses 'object' as the parameter, I didn't write that code, but it looks to be because that is what ApplicationData.Current.LocalSettings.Values.TryGetValue returns so it is preserving the same data type as the settings API gives. I'm inclined to leave it alone and keep the warning suppress since it matches the windows API it is using.